src/typeahead/typeahead.spec.ts

```diff
@@ -496,7 +496,7 @@ describe('ngb-typeahead', () => {
 
       it('should restore hint when results window is dismissed', async(() => {
            const fixture = createTestComponent(
-               `<input type="text" [(ngModel)]="model" [ngbTypeahead]="findAnywhere" [showHint]="true"/>`);
+               `<input type="text" [(ngModel)]="model" [ngbTypeahead]="findAnywhere" [showHint]="true" [autoSelectItem]="true" />`);
```

Comment:
We should _not_ modify existing tests. `autoSelectItem` should be default. Revert.